lib/s3_inventory.rb

```diff
@@ -0,0 +1,187 @@
+require "aws-sdk-s3"
+require "csv"
+
+class S3Inventory
```

Comment:
I think we need to test this class as much as possible. Otherwise, it is hard for the PR to be reviewed.